qa-pipelines/tasks/cf-upgrade.sh

```diff
@@ -48,10 +48,26 @@ instance_count=$(kubectl get statefulsets -o json diego-cell --namespace scf | j
 monitor_file=$(mktemp -d)/downtime.log
 monitor_url "http://go-env.${DOMAIN}" "${monitor_file}" &
 
+pxc_post_upgrade() {
```

Comment:
Just `[[ "${HA}" == true ]]` in this function would be sufficient, and also clearer that this function is just an alias for this check (as opposed to the inversion or something)

Also a nit: the indentation here doesn't match the rest of the file